src/role-access/containers/exclusion/exclusion-navigation/exclusion-navigation.component.html

```diff
@@ -0,0 +1,26 @@
+<a class="govuk-back-link" href="javascript:void(0)"
+   *ngIf="isVisible((navigationCurrentState$ | async), backVisibilityStates)"
+   (click)="onEventTrigger(exclusionNavigationEvent.BACK)">Back</a>
+<div>
```

Comment:
**Nitpick**  Will do need the extra `div` to wrap `ng-content`

Reply:
removed div